xml/deployment_prep_aarch64_raspi.xml

```diff
@@ -323,7 +355,7 @@ dtoverlay=i2c-rtc,ds1307</screen>
   </para>
   <variablelist>
    <varlistentry>
-    <term>&sls; 15 SP3 Release Notes</term>
+    <term>&sls; 15 SP4 Release Notes</term>
```

Comment:
It would be advisable to keep that in a separate commit, to facilitate your backporting to SP3 (and any SLE Micro 5.x releases based on SP3). Currently combined with addition of RPi4. Could even be done in a separate SR to not be blocked by text discussion.